Update Web Forms docs now that it is the default in Central (#2072)
opendatakit/docs: docs/web-forms-intro.rst

.. spelling:word-list::
    yyyy

ODK Web Forms
====================

ODK Central provides a web-based interface to your forms for data edits, to preview form definitions, and to enable data collection from devices other than Android phones. You can learn about how to collect data using web forms in :ref:`the section on managing Submissions <central-web-submissions>`.

> ODK is in the process of migrating to ODK Web Forms, a modern web forms experience. It is the default for new Forms starting in Central v2026.2.0. Forms created prior to that use `Enketo <https://enketo.org/>`_ by default.

Reviewer: ```suggestion
ODK is migrating to ODK Web Forms, a modern web forms experience, and is the default for new Forms starting in Central v2026.2.0. Forms created prior to that will continue to use `Enketo <https://enketo.org/>`_.
```

Reviewer: Alternatively change "use Enketo" to "used Enketo".

> This page describes how to :ref:`opt a Central Form into using ODK Web Forms <web-forms-opt-in>` for all web-based functions, outlines :ref:`features not yet supported <features-not-yet-supported>` and documents :ref:`more complex question types <web-forms-question-types>`.

Reviewer: ```suggestion
This page describes how to :ref:`opt into using ODK Web Forms <web-forms-opt-in>` for all web-based functions, outlines :ref:`features not yet supported <features-not-yet-supported>` and documents :ref:`more complex question types <web-forms-question-types>`.
```

.. _web-forms-opt-in:

Opting into ODK Web Forms in Central
----------------------------------------

ODK Web Forms is the default web forms experience for forms created in Central v2026.2.0 or later. Forms created on prior versions used `Enketo <https://enketo.org/>`_ by default.

To opt an existing form into ODK Web Forms, go to the :guilabel:`Settings` tab for that form. In the :guilabel:`Web forms experience` section, select the :guilabel:`ODK Web Forms` option, read the description, and confirm that you want to opt in.

You can change a form's setting between Enketo and Web Forms at any time. Any form links that users already have will continue to work and will reflect the setting on the server at the time that the form link is loaded. Existing submissions are not affected by switching between web forms options.

.. _features-not-yet-supported:

Features not yet supported
---------------------------

We've extensively tested ODK Web Forms and consider it ready for production use. However, there is some advanced functionality it does not yet support. Here is functionality in Enketo that we intend to add to Web Forms, in rough order of priority:

> * One question per page

Reviewer: I wonder if this list should be dynamic, eg: link to GH Issues with a filter for a tag for "available in Enketo"? This has two advantages: a) we don't need to update this documentation every release, and b) users who want a specific feature can watch and/or give feedback on the issue.

Author: Here's what that might look like: https://github.com/getodk/web-forms/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22In%20Enketo%22

I'd rather have something inline because it's so much easier to skim. Maybe we could [link to the list of issues tagged as features](https://github.com/getodk/web-forms/issues?q=is%3Aissue%20state%3Aopen%20type%3AFeature)? We generally try to encourage end users to share on the forum though!

Reviewer: I like the GH list but I'm not the target audience. I leave it up to you.
